src/main/java/net/dries007/tfc/common/blocks/devices/TFCComposterBlock.java

```diff
@@ -131,6 +131,18 @@ public boolean isRandomlyTicking(BlockState state)
         return state.getValue(TYPE) != CompostType.ROTTEN;
     }
 
+    @Override
+    public int getAnalogOutputSignal(BlockState state, Level level, BlockPos pos)
+    {
+        return state.getValue(TYPE).ordinal();
```

Comment:
imo this should output `STAGE`, `not `TYPE` ? I understand that makes it harder to automate, but it *is* more inline with what comparators do - measure the full-ness of things.